test/e2e-test/src/index.ts

```diff
@@ -499,18 +499,6 @@ class Demo {
       keyResponse.gt["x-ms-ver"] === "1",
     );
 
-    // JWT not allowed
-    [statusCode, keyResponse] = await Api.keyReleasePolicy(
-      this.demoProps,
-      member,
```

Comment:
use a cert instead of deleting the test

Reply:
My understanding of this test is that we're checking that if you call `/keyReleasePolicy` with JWT auth it should fail, but this property is no longer the case.

It isn't the case for user cert either so what would changing to a user cert achieve?